File: admin-panel/src/config/app.config.ts
export const AppConfig = {
  appName: 'RideR Admin',
  apiBaseUrl: import.meta.env.VITE_API_BASE_URL || 'http://localhost:3000/api/v1',
  socketUrl: import.meta.env.VITE_SOCKET_URL || 'http://localhost:3000',
  tokenKey: 'admin_token',
  refreshTokenKey: 'admin_refresh_token',
  pageSize: 10,
  dateFormat: 'DD/MM/YYYY',
  timeFormat: 'HH:mm:ss',
};

File: driver-app/lib/src/core/config/app_config.dart
import 'package:flutter_dotenv/flutter_dotenv.dart';

class AppConfig {
  static String get appName => 'RideR Partner';
  static String get apiBaseUrl => dotenv.env['API_BASE_URL'] ?? 'http://localhost:3000/api/v1';
  static String get socketUrl => dotenv.env['SOCKET_URL'] ?? 'http://localhost:3000';
  static String get appVersion => '1.0.0';
  static String get appPlatform => 'driver';
  static const int requestTimeout = 30000;
  static const int locationUpdateInterval = 5000;
  static const double defaultMapLatitude = 19.0760;
  static const double defaultMapLongitude = 72.8777;
  static const double defaultMapZoom = 14.0;
}
